src/servers/cron.ts

```diff
@@ -42,6 +43,11 @@ const main = async () => {
     await deleteExpiredInvoiceUser()
   }
 
+  const updateLnPaymentsCollection = async () => {
+    const result = await updateLnPayments()
+    if (result instanceof Error) throw result
```

Comment:
should it be a throw or return here @dolcalmi ?

Reply:
Relevant comment: https://github.com/GaloyMoney/galoy/pull/848#discussion_r784882767